std/protocols/permutation.asm

```diff
@@ -53,12 +54,11 @@ let compute_next_z: Fp2<expr>, Fp2<expr>, Fp2<expr>, Constr -> fe[] = query |acc
 };
 
 /// Returns constraints that enforce that lhs is a permutation of rhs
+/// WARNING: This function can currently not be used multiple times since
+/// the used challenges would overlap
+/// TODO: Implement this for an array of constraints
 ///
 /// # Arguments:
-/// - acc: A phase-2 witness column to be used as the accumulator. If 2 are provided, computations
-///        are done on the F_{p^2} extension field.
-/// - alpha: A challenge used to compress the LHS and RHS values
-/// - beta: A challenge used to update the accumulator
 /// - permutation_constraint: The permutation constraint
```

Comment:
The `Arguments` part of the documentation provides no information, so I think we can just delete it.